setup.php

```diff
@@ -78,6 +78,11 @@ function plugin_init_actualtime() {
       $PLUGIN_HOOKS['add_javascript']['actualtime'] = 'js/actualtime.js';
       $PLUGIN_HOOKS['item_purge']['actualtime']=['TicketTask'=>'plugin_actualtime_item_purge'];
 
+      if (strpos($_SERVER['REQUEST_URI'], 'ticket.form.php') !== false) {
+         $ajax_url = Plugin::getWebDir('actualtime') . '/ajax/timer.php';
+         echo Html::scriptBlock("document.addEventListener('DOMContentLoaded', function(){window.actualTime.init('$ajax_url');});");
+      }
+
```

Comment:
With this code the active timer is only shown when you are on the ticket page, the normal operation is that it is shown on all pages